app/actions.py

```diff
@@ -177,6 +177,8 @@ def cursorMove(self, rowDelta, colDelta):
 
   def getCursorMoveAndMark(self, rowDelta, colDelta, markRowDelta,
       markColDelta, selectionModeDelta):
+    if self.penCol + colDelta < 0: #Catch cursor at beginning of line
```

Comment:
Oh yeah sorry, I'm still learning all the styles and stuff, picking stuff up here and there. Should we start doing that from now on? The google python style guideline specifies 2 spaces before a comment so it would be 

if self.penCol + colDelta < 0:  # Catch cursor at beginning of line.

Mix and match? :+1: